resources/services/kinesis/gen.hcl

```diff
@@ -45,9 +45,255 @@ resource "aws" "kinesis" "streams" {
       params = ["stream_arn"]
     }
   }
+  userDefinedColumn "tags" {
+    type              = "json"
+    generate_resolver = true
+  }
+}
 
+resource "aws" "kinesis" "firehoses" {
+  path = "github.com/aws/aws-sdk-go-v2/service/firehose/types.DeliveryStreamDescription"
+  ignoreError "IgnoreAccessDenied" {
+    path = "github.com/cloudquery/cq-provider-aws/client.IgnoreAccessDeniedServiceDisabled"
+  }
+  deleteFilter "AccountRegionFilter" {
+    path = "github.com/cloudquery/cq-provider-aws/client.DeleteAccountRegionFilter"
+  }
+  multiplex "AwsAccountRegion" {
+    path   = "github.com/cloudquery/cq-provider-aws/client.ServiceAccountRegionMultiplexer"
+    params = ["firehose"]
+  }
+  userDefinedColumn "account_id" {
+    description = "The AWS Account ID of the resource."
+    type        = "string"
+    resolver "resolveAWSAccount" {
+      path = "github.com/cloudquery/cq-provider-aws/client.ResolveAWSAccount"
+    }
+  }
+  userDefinedColumn "region" {
+    type        = "string"
+    description = "The AWS Region of the resource."
+    resolver "resolveAWSRegion" {
+      path = "github.com/cloudquery/cq-provider-aws/client.ResolveAWSRegion"
+    }
+  }
+  options {
+    primary_keys = ["arn"]
+  }
   userDefinedColumn "tags" {
     type              = "json"
     generate_resolver = true
   }
+  column "destinations" {
+    skip = true
+  }
+  column "has_more_destinations" {
+    skip = true
+  }
+  userDefinedColumn "arn" {
+    type        = "string"
+    description = "The Amazon Resource Name (ARN) of the delivery stream"
+    resolver "resolveStreamArn" {
+      path   = "github.com/cloudquery/cq-provider-sdk/provider/schema.PathResolver"
+      params = ["DeliveryStreamARN"]
```

Comment:
can you do the 'rename' for the column instead?
```hcl
column "delivery_stream_arn" {
  rename = "arn"
}
```

Reply:
We don't want to do `rename` because we want to preserve all of the data in its original form, we just want to add a new field